clio-link-checking/clio-link-checking-service/src/main/java/eu/europeana/clio/link/checking/service/config/properties/LinkCheckingConfigurationProperties.java

```diff
@@ -0,0 +1,95 @@
+package eu.europeana.clio.link.checking.service.config.properties;
+
+import eu.europeana.clio.link.checking.service.config.Mode;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Class using {@link ConfigurationProperties} loading.
+ */
+@ConfigurationProperties(prefix = "link-checking")
+public class LinkCheckingConfigurationProperties {
```

Comment:
Without a constructor written in here, it allows us to create this object with no defined parameters.
Do we want that?

Reply:
I don't think we can do anything about this since this is how spring boot requires it.